<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerAddressBook</class>
 <widget class="QWidget" name="CustomerAddressBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="InputLayout">
    <item>
     <layout class="QVBoxLayout" name="inputLabels">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="CustomerAddress">
        <property name="text">
         <string>Address:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Account ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Interest</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Key Value</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="lineEdits">
      <item>
       <widget class="QLineEdit" name="NameEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="addressEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="EmailEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="AccountIdEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="PasswordEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="interestEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="keyValueEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="addSubmitCancelLayout">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="submitButton">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>93</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="EditRemoveSearch">
    <item>
     <widget class="QPushButton" name="editButton">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="removeButton">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="searchButton">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="PrevNextButton">
    <item>
     <widget class="QPushButton" name="prevCustomerButton">
      <property name="text">
       <string>Previous Customer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextCustomerButton">
      <property name="text">
       <string>Next Customer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>287</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="toggleAccess">
    <item>
     <widget class="QRadioButton" name="ActivatedCustomer">
      <property name="text">
       <string>Access To Brochure</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>warningLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
